components/Layout/Settings/AddApiKeyDialog.vue

```diff
@@ -2,7 +2,7 @@
   <v-dialog
```

Comment:
I don't think this file should be in layout. It doesn't have any slots. Thoughts @ThibaultNocchi ?